src/DataBus/AzureDataBusPersistence.cs

```diff
@@ -28,6 +29,19 @@ protected override void Setup(FeatureConfigurationContext context)
             context.Services.AddSingleton(blobContainerClientProvider ?? new ThrowIfNoBlobServiceClientProvider());
             context.Services.AddSingleton<IDataBus>(serviceProvider => new BlobStorageDataBus(serviceProvider.GetRequiredService<IProvideBlobServiceClient>(),
                 dataBusSettings, new AsyncTimer()));
+            
+            context.Settings.AddStartupDiagnosticsSection(
+                "NServiceBus.Databus.AzureBlobStorage",
```

Comment:
I'm wondering if we should use typeof(AzureBlobStorage).FullName instead of a string. This is not a hot path. But I'll let @danielmarbach chime in.

Reply:
In the persistences, we use a hard-coded section name that represents the name of the package. So this is aligned with that. If we create sections in multiple places we might think about adding a constant but I guess this is fine right now